### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by KareemWilson**
### In this branch, we did the following: 

- Created a dynamic menu for mobile-view 
- Used Javascript and Document Object Model "DOM" to make it interactive
- Played with ClassNames by adding and removing them from elements

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -16,11 +16,12 @@
...
-            <li class="nav-item"><a href="#About">About</a></li>
-            <li class="nav-item"><a href="#Contact">Contact</a></li>
+            <li class="nav-item close-menu"><a href="#Welcome">Portofolio</a></li>
+            <li class="nav-item close-menu"><a href="#About">About</a></li>
+            <li class="nav-item close-menu"><a href="#Contact">Contact</a></li>
           </ul>
-          <nav class="fa fa-bars"></nav>
+          <nav class="fa fa-bars anime-x openMenu"></nav>
+          <i class="fa fa-close close-menu invisible"></i>
         </header>
```
Comment (muneebulrehman) [inline index.html]: When I click on a nav link in desktop mode, the nav links disappear. Also, when I close the mobile navbar and go back to desktop mode, the navbar is not visible. Kindly fix this issue and then submit it for a re-review.

File: index.html [hunk 2]
```diff
@@ -16,11 +16,12 @@
         <header class="header">
           <p class="logo">Welcome</p>
           <ul class="nav-list">
-            <li class="nav-item"><a href="#Welcome">Portofolio</a></li>
-            <li class="nav-item"><a href="#About">About</a></li>
-            <li class="nav-item"><a href="#Contact">Contact</a></li>
+            <li class="nav-item close-menu"><a href="#Welcome">Portofolio</a></li>
```
Comment (muneebulrehman) [inline index.html]: When I click on the `portfolio` link in the navbar, it doesn't exactly go the the `portfolio` section. Kindly fix this issue and then submit it for a re-review.

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi @KareemWilson , 

Good job so far! I see you put a lot of effort into this project. However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors.
- Descriptive PR

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by KareemWilson** [comment]
This time I fix this bug and navigate the portfolio button to the "Works Section", and made a new nav-list specialized for mobile-view.

Thank you

**Review by addod19 — APPROVED** [review]
Hi @KareemWilson, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### Highlights
- [ ] Nice job fixing the navigation llinks, now we are redirected to the appropriate part of the page

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**